openprescribing/frontend/tests/test_spending_views.py

```diff
@@ -107,6 +107,17 @@ def test_spending_views(self):
                 self.months[-1]
             )
 
+    def test_alert_signup(self):
+        url = '/practice/{}/concessions/'.format(self.practice.code)
+        data = {
+            'email': 'alice@example.com',
+            'practice': self.practice.code,
+            'newsletters': ['alerts'],
+        }
+        response = self.client.post(url, data, follow=True)
+        self.assertContains(
+            response, "Check your email and click the confirmation link")
+
```

Comment:
Can we test a `NCSOConcessionBookmark` is created here? Possibly also with the optional newsletter signup too?